src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $transition-duration;
+
+    &:hover {
+      color: #00acdc;
+    }
```

```suggestion
    transition: color $transition-duration;

    &:hover {
      color: #00acdc;
    }
```
Необходимо явно указывать свойства, которые будут анимированы, иначе используется transition: all
transition: all достаточно требователен к ресурсам браузера и на старых компьютерах может ухудшать пользовательский опыт